lib/Eclair/LLVM/SymbolTable.hs

```diff
@@ -23,23 +23,22 @@ data SymbolTable
 data CGState
   = CGState
   { symbolTableTy :: Type
-  , symbolTy :: Type
```

Comment:
I don't like that we need to rename our field names just to silence a warning. Can we turn off unused warning for this file? (With a `{-# OPTIONS_GHC ... #-}`